pymatviz/utils.py

```diff
@@ -300,25 +300,26 @@ def annotate_metrics(
 def crystal_sys_from_spg_num(spg: int) -> CrystalSystem:
     """Get the crystal system for an international space group number."""
     if not isinstance(spg, int):
-        raise TypeError(f"Expect integer for space group number, got {type(spg)}")
+        raise TypeError(
+            f"Expect integer for space group number, got {type(spg).__name__}"
+        )
 
-    if spg < 1 or spg > 230:
-        raise ValueError(f"Invalid space group number {spg}")
+    if not (1 <= spg <= 230):
+        raise ValueError(f"Invalid space group number {spg}, must be 1 <= num <= 230")
 
-    elif 1 <= spg <= 2:
+    if 1 <= spg <= 2:
         return "triclinic"
-    elif spg <= 15:
+    if spg <= 15:
         return "monoclinic"
-    elif spg <= 74:
+    if spg <= 74:
         return "orthorhombic"
-    elif spg <= 142:
+    if spg <= 142:
         return "tetragonal"
-    elif spg <= 167:
+    if spg <= 167:
         return "trigonal"
-    elif spg <= 194:
+    if spg <= 194:
```

Comment:
Interesting change here... I know this should work exactly the same, but I'm just curious why preferring a chain of `if` over `elif`?

For me I prefer `elif` for two reasons:
1. It's a straight forward indicator that exactly one branch would be picked.
2. In other cases where the loop is not broken directly by `return`, use a chain of `if` would lead to the following statements still being assessed even when the former condition has already be met, discussed [here](https://stackoverflow.com/questions/23230044/if-and-elif-chain-versus-a-plain-if-chain), for example:
```python
a = 1

if a > 0:  # this would be assessed
    print("a greater than zero!")
if a < = 0:  # this would also be assessed
    print("a not greater than zero!")
```

Reply:
sorry, forgot to respond.

i'd be happy to use `elif` too but `ruff` complains and didn't disagree enough to want to override `ruff`'s config

Reply:
Maybe I would see if I could tag `ruff` to ignore this part in the following PR! Thanks!